nxviz/geometry.py

```diff
@@ -24,6 +24,30 @@ def node_theta(nodelist, node):
     return theta
 
 
+def group_theta(node_length, node_idx):
+
+    theta = -np.pi + node_idx*2*np.pi/node_length
+    return theta
+
+
+def text_alignement(x, y):
+
```

Comment:
This is definitely a great function to have! Let me take a stab at the necessary docstring below, could you modify it and add it to the function please?

```python

def text_alignment(x, y):
    """
    Align text labels based on the x- and y-axis coordinate values.

    This function is used for computing the appropriate alignment of the text 
    label. 

    For example, if the text is on the "right" side of the plot, we want it to 
    be left-aligned. If the text is on the "top" side of the plot, we want it
    to be bottom-aligned. 

    :param x, y: (`int` or `float`) x- and y-axis coordinate respectively.
    :returns: A 2-tuple of strings, the horizontal and vertical alignments
        respectively.
    """

Reply:
Ah and alignment is misspelled!